lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/JavaBridge.java

```diff
@@ -28,12 +30,14 @@
 @Interop(CppFiles = "JavaClrBridge.cs")
 public class JavaBridge {
   private static final String CPP_BRIDGE = "JavaClrBridge";
+  private static final Logger LOG = Logger.getLogger(JavaBridge.class.toString());
 
   static {
     try {
       System.loadLibrary(CPP_BRIDGE);
     } catch (final UnsatisfiedLinkError e) {
       // TODO[JIRA REEF-383] Document/Refactor JavaBridge
+      LOG.severe("TODO[JIRA REEF-383] Cannot load native JavaClrBridge.");
```

Comment:
I don't think we need to log TODO; it's message not to the user, but to the devs.